### DISCUSSION TITLE
Total Transactions per User vs. Minutes for Test Duration

### DISCUSSION THREAD

**Discussion by siddhsql** [TPC-Council/HammerDB#544] [General]
https://www.hammerdb.com/docs/ch04s06.html
am i correct in understanding that the test will run for Minutes for Test Duration. during this time each virtual user will execute as many transactions as they possibly can up to Total Transactions per User? why have Total Transactions per User at all?

**Comment by sm-shaw** [comment]
Why have Total Transactions per User at all? HammerDB is very flexible and supports many use case scenarios, so you might want the virtual users to log on and off during the test. In the GUI this will increment the iterations counter.
